### ISSUE TITLE
Support TypeScript 5

### ISSUE THREAD

**Issue by ntucker** [sandersn/downlevel-dts#88]
https://github.com/microsoft/TypeScript/wiki/API-Breaking-Changes#typescript-50

**Comment by hanayashiki** [comment]
If this library doesn't work with ts5, how it specifies its dependency as 'next'?

https://github.com/sandersn/downlevel-dts/blob/master/package.json#L21

I noticed this because every day I try to update my dependencies, I download a latest build of typescript for nothing.

**Comment by dannolan** [comment]
Yeah noting the above we are constantly downloading the nightly of typescript

**Comment by nbilyk** [comment]
I made a fork that addresses several issues with more recent versions of typescript as well as improving the core transformation logic.  I've been using it successfully in large projects.

https://www.npmjs.com/package/downlevel-dts
https://github.com/nbilyk/downlevel-dts

If the original author sees this I'd be happy to see this work make it into the original project.
